test/e2e/network/netpol/model.go

```diff
@@ -279,15 +280,21 @@ func (c *Container) Spec() v1.Container {
 	case v1.ProtocolUDP:
 		cmd = []string{"/agnhost", "serve-hostname", "--udp", "--http=false", "--port", fmt.Sprintf("%d", c.Port)}
 	case v1.ProtocolSCTP:
-		cmd = []string{"/agnhost", "netexec", "--sctp-port", fmt.Sprintf("%d", c.Port)}
+		env = append(env, v1.EnvVar{
+			Name:  fmt.Sprintf("SERVE_SCTP_PORT_%d", c.Port),
+			Value: "foo",
+		})
+		cmd = []string{"/agnhost", "porter"}
```

Comment:
This is cool, never tried `porter` before -- what does it do?

Reply:
Porter is the default method used by `networkpolicy/` tests, you set a variable like SERVE_SCTP_PORT_80|81 and it uses the proto/port from the variables instead of arguments in the command line:

https://github.com/kubernetes/kubernetes/blob/fec1a366c3b17b5d46b79ddac0b8bb04dfd212ee/test/e2e/network/network_policy.go#L2024

One issue with `netexec` is even if you pass only the `--sctp-port` a TCP 8080 and UDP 8081 are used, if you change the sctp port in another container it will die with a bind address issue, to reduce the complex I change it to `porter` when SCTP is used:

for xref: https://kubernetes.slack.com/archives/C09QZ4DQB/p1610316853393700

Reply:
👍